cake-recipe.txt

```diff
@@ -16,7 +16,7 @@ Directions:
     pans.
  
  2. In a large bowl, combine the sugar, flour, cocoa, baking powder, baking soda
-    and salt.  Add eggs, yoghurt, oil and vanilla. Mix until smooth.  Stir in
+    and salt.  Add eggs, milk, oil and vanilla. Mix until smooth.  Stir in
```

Comment:
No error but this is just for fun. Make changes like this in two different commits next time.